source/2020-03-16-ember-3-17-released.md

```diff
@@ -91,7 +95,19 @@ There are no new deprecations in Ember CLI 3.17.
 ---
 
 For more details on the changes in Ember CLI 3.17 and detailed upgrade
-instructions, please review the [Ember CLI  3.17.0 release page](https://github.com/ember-cli/ember-cli/releases/tag/v3.17.0).
+instructions, please review the [Ember CLI 3.17.0 release page](https://github.com/ember-cli/ember-cli/releases/tag/v3.17.0).
+
+## Ember 3.16 is an LTS Release
+
+If you are upgrading from the previous LTS version (3.12) to 3.16 then you are getting access to these features:
+
+- Tracked properties (3.13)
```

Comment:
I looked at 3.12-3.15 release notes to gather features that impact app developers (and omitted features for addon developers).

Reply:
Really awesome, thanks!